platform/ext/common/gcc/tfm_common_s.ld

```diff
@@ -602,6 +602,29 @@ SECTIONS
 
 #endif /* TFM_LVL != 1 */
 
+    .ER_TFM_CODE : ALIGN(4)
```

Comment:
I tried without the tfm_common.ld commit, and it works fine for me.

Reply:
I could not build either Regression or RegressionIPC configuration (using gcc 7.3.1) without tweaking the .ld file. Same when building the Zephyr samples (gcc 9.2.0 there), the secure image just did not fit in FLASH without the change.
But since it works for you, perhaps I just need to correct something in my setup.

Reply:
Maybe you could try once more with the commit completely removed? If you have problems to do with the gcc version we need to accommodate that since there are multiple valid gcc versions.

Reply:
I checked once more. Indeed, I can build the Regression configuration without those changes in the linker script (I'm not sure why it didn't work for me earlier), but not the RegressionIPC one. Have you tried to build it?

Reply:
No, I haven't tried RegressionIPC. We might need the change after all then.

Reply:
I checked now and ConfigRegressionIPC fails for me as well.